internal/pkg/service/buffer/sink/tablesink/storage/level/local/file_test.go

```diff
@@ -27,7 +27,7 @@ func TestFile_Validation(t *testing.T) {
 				Dir:            "my-dir",
 				Compression:    compression.NewConfig(),
 				DiskSync:       disksync.NewConfig(),
-				DiskAllocation: allocate.NewConfig(),
+				DiskAllocation: diskalloc.NewConfig(),
```

Comment:
I renamed package `allocate` -> `diskalloc` to match `disksync` naming, and make the name self explaining.